### ISSUE TITLE
ERROR: using TypeScript error, support please

### ISSUE THREAD

**Issue by soongyu** [jscottsmith/react-scroll-parallax#77]
Currently, I've face an error while using a library called React-Scroll-Parallax. It had no problem with JS but TS kept saying to me like as follows:

```
const SubBanner: React.FC<SubBannerProps> = ({ layers, style }) => {
  return (
    <Wrapper>
      <ParallaxProvider>
        <ParallaxBanner
          layers={[
            {
              image:
                'assets/images/main-image.png',
              amount: 0.5,
            },
          ]}
          style={{
            height: '300px',
          }}
          />
      </ParallaxProvider>
    </Wrapper>
  );
};
```

```
Property 'children' is missing in type '{ image: string; amount: number; }' but required in type 'BannerLayer'.  TS2741
```

It seems this library doesn't support the current TS, so how to solve the problem?? I need to use it with TS.

**Comment by KyorCode** [comment]
Did you manage to solve this?

**Comment by newerton** [comment]
Solution?

**Comment by jscottsmith** [comment]
Published fix in [`2.3.3` ](https://github.com/jscottsmith/react-scroll-parallax/releases/tag/v2.3.3)
